spec/pack_stats_spec.rb

```diff
@@ -641,7 +631,6 @@ module PackStats # rubocop:disable RSpec/DescribedClassModuleWrapping
             modularization.all_packages.with_violations.count
             modularization.by_package.all_files.count
             modularization.by_package.dependency_violations.count
-            modularization.by_package.inbound_dependency_violations.count
```

Comment:
I added this test on `main`. Its purpose is to show the unique metric names that pack_stats emits.

More info at https://github.com/rubyatscale/pack_stats/pull/35/commits/4603dceafe782a8a7101f56a8cad374392a1f943#diff-8977ec6cc28cca4acf977af397d9b00d83eeede5b1a33de4d19d2850e0c0bd71R594

You can look at changes to these test expectations to get a summary of which metrics are being changed how.

You can look at changes to the other specific metric tests to see exactly how definitions are being changed.